src/theme.rs

```diff
@@ -656,20 +656,27 @@ pub trait Theme {
 
     /// Returns a log message rendering with a chosen symbol.
     fn format_log(&self, text: &str, symbol: &str) -> String {
+        self.format_log_with_spacing(text, symbol, true)
+    }
+
+    /// Returns a log message rendering with a chosen symbol, with an optional trailing empty line
+    fn format_log_with_spacing(&self, text: &str, symbol: &str, spacing: bool) -> String {
```

Comment:
@beeb After removing the argument from the `println` method, this change no longer looks necessary. Seems like, `theme.rs` should go unchanged in this PR.

Reply:
The problem is that the original implementation of `format_log` automatically adds an empty line between log entries, which is not desirable (in my opinion) for the case of this `println` function.